### PR TITLE
feat: Increase accuracy of Chromium Android detection

### PR DESCRIPTION
**Pull request by dj-stormtrooper**

### CODE REVIEW

File: src/utils/browser.ts [hunk 1]
```diff
@@ -281,23 +281,30 @@ export function exitFullscreen(): Promise<void> {
...
-  return (
-    countTruthy([
-      'onorientationchange' in w,
-      'orientation' in w,
-      isItChromium && !('SharedWorker' in w),
-      isItGecko && /android/i.test(navigator.appVersion),
-    ]) >= 2
-  )
+  if (isItChromium) {
+    return countTruthy([!('SharedWorker' in w), isAndroidNotificationError(), 'sinkId' in new window.Audio()]) >= 2
```
Comment (bayotop) [inline src/utils/browser.ts:289]: This should be `!('sinkId' in new window.Audio())`, because it's not available on Android.

Comment (dj-stormtrooper) [reply src/utils/browser.ts]: Good catch, thanks!

**Comment by dj-stormtrooper** [comment]
> Please download some old Chrome and Firefox version APKs and check the detector function on an Android Emulator or a real device. One good APK source is https://4pda.to/forum/.

Tested on Firefox 67 and Chrome 91, works fine

**Review by Finesse — APPROVED** [review]

**Review by bayotop — APPROVED** [review]

**Comment by Finesse** [comment]
@dj-stormtrooper Could you please update [the Android detection function in BotD](https://github.com/fingerprintjs/BotD/blob/5d6544f24278fad2c9761001f4436410381a9bb3/src/utils/browser.ts#L75) too?

**Pull request merged**